resources/js/electron-plugin/src/server/php.ts

```diff
@@ -165,7 +165,7 @@ function getDefaultEnvironmentVariables(secret, apiPort) {
     NATIVEPHP_STORAGE_PATH: storagePath,
     NATIVEPHP_DATABASE_PATH: databaseFile,
     NATIVEPHP_API_URL: `http://localhost:${apiPort}/api/`,
-    NATIVEPHP_RUNNING: true,
+    NATIVEPHP_RUNNING: 'true',
```

Comment:
This change feels weird to me... can you explain why a boolean is suddenly a string and how this ends up being ok?